dotcom-rendering/src/client/index.apps.ts

```diff
@@ -0,0 +1,36 @@
+import './webpackPublicPath';
+
+// these modules are bundled in the initial (i.e. this) chunk, so that they run ASAP
+import { dynamicImport } from './dynamicImport';
+import { ga } from './ga';
+import { islands } from './islands';
+import { recordInitialPageEvents } from './ophan/recordInitialPageEvents';
+import { performanceMonitoring } from './performanceMonitoring';
+import { sentryLoader } from './sentryLoader';
+import { startup } from './startup';
+
+startup('recordInitialPageEvents', recordInitialPageEvents);
+startup('ga', ga);
```

Comment:
not necessarily for this PR, but I'm not sure if apps articles would still have Google Analytics. At least, I don't think AR loads GA.